charts/identity-platform/values.yaml

```diff
@@ -660,6 +657,16 @@ ds_snapshot:
     pullPolicy: IfNotPresent
     imagePullSecrets: []
 
+ds_restore:
+    # Restore a backup taken by the dsbackup tool
```

Comment:
ds-backup.sh script tool ?

Reply:
no dsbackup is the ds tool on the pod. ds-backup.sh calls this tool